core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-ema-bookmarks/dot-ema-bookmarks.component.html

```diff
@@ -0,0 +1,7 @@
+<p-button
```

Comment:
When `button` pass from loading to finish it jumps because the loading icon is smaller than the icon:

```
    display: flex;
    width: 20px;
    height: 20px;
    align-items: center;
    justify-content: center;
```

We need to add something like this in the `_button`

Reply:
Done!